shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2319,22 +2326,64 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     CGFloat pointerY = (CGFloat)[args[@"pointerY"] doubleValue];
     [self handlePointerMove:pointerY];
     result(nil);
+  } else if ([method isEqualToString:kOnInteractiveKeyboardPointerUpMethod]) {
+    CGFloat pointerY = (CGFloat)[args[@"pointerY"] doubleValue];
+    [self handlePointerUp:pointerY];
+    result(nil);
   } else {
     result(FlutterMethodNotImplemented);
   }
 }
 
+- (void)handlePointerUp:(CGFloat)pointerY {
+  // View must be loaded at this point.
+  UIScreen* screen = _viewController.flutterScreenIfViewLoaded;
+  CGFloat screenHeight = screen.bounds.size.height;
+  CGFloat keyboardHeight = _keyboardRect.size.height;
+  BOOL pointerBelowMiddleY = (screenHeight - (keyboardHeight / 2)) < pointerY;
```

Comment:
I would name this `shouldDismissKeyboard` or similar. Ideally describe the intent of the variable rather than how it's computed (which is already on the right-hand side here).